_layouts/people.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "People Involved in the Conference"
+---
+
+<div style="margin-top:2em;">
```

Comment:
```suggestion
<div class="columns">
  <div class="column col-10 col-sm-12">
```
Could you please try this, for responsive design. Will need an additional closing `</div>` at the end.